go.mod

```diff
@@ -2,66 +2,46 @@ module github.com/matrix-org/dendrite
 
 require (
```

Comment:
Could you remove `go.mod` and `go.sum` from this PR please? This PR shouldn't require an update to the dependencies.